src/errors/test-run/templates.js

```diff
@@ -451,4 +452,8 @@ export default {
     [TEST_RUN_ERRORS.actionFunctionOptionError]: err => `
         The value of the "${err.optionName}" option belongs to an unsupported data type (${err.actualValue}). The "${err.optionName}" option only accepts function types values.
     `,
+
+    [TEST_RUN_ERRORS.actionInvalidObjectPropertyError]: err => `
+        The "${err.objectName}" object contains an invalid property: "${err.propertyName}". Please use the following properties to configure "${err.objectName}": ${getConcatenatedValuesString(err.availableProperties)}.
```

Comment:
I propose to use the following scheme for this error message:
- why did this happen
- what i should do to fix it

```js
The "${err.objectName}" object contains an invalid property: "${err.propertyName}". You can safely remove it due to this property is not applied.  Make sure you have correctly specified the action options. List of available options for the "${err.objectName}" are: ${getConcatenatedValuesString(err.availableProperties)}.
```

Reply:
Also, I see you removed the `rootProp.nestedProp` way for option specifying. Please, add a test that checks the invalid nested property. I guess we should display the full path to the property.